script/export-components

```diff
@@ -33,15 +33,25 @@ main() {
   if [[ -z "${version}" ]]; then
     fail "unable to determine version"
   fi
+  info "exporting components with version ${version}"
 
   cd "${tuf_dir}"
   tuf clean
-  cp "${ROOT}/host/bin/flynn-host.gz"                      "staged/targets/flynn-host.gz"
-  gzip -9 --stdout "${ROOT}/host/bin/flynn-init"         > "staged/targets/flynn-init.gz"
-  gzip -9 --stdout "${ROOT}/host/bin/flynn-nsumount"     > "staged/targets/flynn-nsumount.gz"
-  gzip -9 --stdout "${ROOT}/host/upstart.conf"           > "staged/targets/upstart.conf.gz"
-  gzip -9 --stdout "${ROOT}/bootstrap/bin/manifest.json" > "staged/targets/bootstrap-manifest.json.gz"
-  gzip -9 --stdout "${ROOT}/version.json"                > "staged/targets/version.json.gz"
+
+  # add the flynn-host binary at the top level so it can be found by the install script
+  cp "${ROOT}/host/bin/flynn-host.gz" "staged/targets/flynn-host.gz"
```

Comment:
No, this is just so that the install script can [download a temporary flynn-host](https://github.com/flynn/flynn/blob/release-channels/script/install-flynn.tmpl#L117-L123), both `flynn-host downlaod` (in the install script) and `flynn-host update` download binaries based on the configured channel.